packages/renderer/src/lib/kube/details/KubeServiceArtifact.svelte

```diff
@@ -29,18 +36,12 @@ export let artifact: V1ServiceSpec | undefined;
     <Cell>Session Affinity</Cell>
     <Cell>{artifact?.sessionAffinity}</Cell>
   </tr>
-  {#if artifact.ports}
-    <tr>
-      <Cell>Ports</Cell>
-      <Cell>
-        {#each artifact.ports as port}
-          <div>
-            {port.name ? port.name + ':' : ''}{port.port}{port.nodePort ? ':' + port.nodePort : ''}/{port.protocol}
-          </div>
-        {/each}
-      </Cell>
-    </tr>
-  {/if}
+  <KubePorts namespace={namespace} resourceName={serviceName} kind={WorkloadKind.SERVICE} ports={artifact.ports?.map((port) => ({
+    name: port.name,
+    value: port.port,
+    protocol: port.protocol,
+    displayValue: `${port.name ? port.name + ':' : ''}${port.port}${port.nodePort ? ':' + port.nodePort : ''}/${port.protocol}`,
```

Comment:
nitpick: I think we should have helper to transform a port object to a UI object  ( having a displayName). And in the svelte component, just display the display name field, not try to have condition / concatenation